lldb/tools/lldb-dap/Protocol/ProtocolRequests.h

```diff
@@ -242,6 +242,8 @@ struct Configuration {
   std::string platformName;
 };
 
+enum Terminal : unsigned { eConsole, eIntegrated, eExternal };
```

Comment:
I believe the format for these should be `eTerminalConsole`, `eTerminalIntegrated`, `eTerminalExternal`.